Assets/Scripts/SitelinkMarker.cs

```diff
@@ -81,7 +81,14 @@ public class SitelinkMarker : MonoBehaviour, IPointerEnterHandler, IPointerExitH
         lineColor.a = blinkLevel;
 
         // Redraw to update blink level.
-        Draw();
+        if (fromSiteMarker.gameObject.layer != 12 && toSiteMarker.gameObject.layer != 12)
+        {
+            Draw();
```

Comment:
Also just making sure you know that not calling this doesn't stop the sitelink line from being drawn, it just stops updating its transparency. To make it actually disappear you probably have to set the game object attached to the marker as not active.